src/main/java/com/ezasm/simulation/Register.java

```diff
@@ -68,7 +68,9 @@ public double getDouble() {
     public void setData(RawData data) {
         if (number != 0) {
             System.arraycopy(data.data(), 0, this.data.data(), 0, this.data.data().length);
-            Window.getInstance().getRegisterTable().addHighlightValue((int) this.number);
+            if (Window.getInstance() != null) {
```

Comment:
We have a function in Window that checks this condition: try `if (Window.hasInstance())`